PLUGINS.md

```diff
@@ -1,4 +1,35 @@
-# Linea plugins 
+# Linea plugins
+
+## Shared components
+### Profitability calculator
+The profitability calculator is a shared component, that is used to check if a tx is profitable.
+It is applied, with different configuration to:
+1. `linea_estimateGas` endpoint
+2. Tx validation for the txpool
+3. Tx selection during block creation
+
+#### CLI Options
+
+| Option Name              | Default Value | Command Line Argument                     |
+|--------------------------|---------------|-------------------------------------------|
+| L1_VERIFICATION_GAS_COST | 1_200_000     | `--plugin-linea-verification-gas-cost`    |
+| L1_VERIFICATION_CAPACITY | 90_000        | `--plugin-linea-verification-capacity`    |
+| L1_L2_GAS_PRICE_RATIO    | 15            | `--plugin-linea-gas-price-ratio`          |
+| L2_GAS_PRICE_ADJUSTMENT  | 0 wei         | `--plugin-linea-gas-price-adjustment`     |
+| MIN_MARGIN               | 1.0           | `--plugin-linea-min-margin`               |
+| ESTIMATE_GAS_MIN_MARGIN  | 1.0           | `--plugin-linea-estimate-gas-min-margin`  |
+| TX_POOL_MIN_MARGIN       | 0.5           | `--plugin-linea-tx-pool-min-margin`       |
+| UNPROFITABLE_CACHE_SIZE  | 100_000       | `--plugin-linea-unprofitable-cache-size`  |
+| UNPROFITABLE_RETRY_LIMIT | 10            | `--plugin-linea-unprofitable-retry-limit` |
+
+### L1 L2 Bridge
+
+#### CLI Options
+
+| Option Name                      | Default Value | Command Line Argument                       |
+|----------------------------------|---------------|---------------------------------------------|
+| L2L1_BRIDGE_CONTRACT_ADDRESS     |               | `--plugin-linea-l1l2-bridge-contract`       |
```

Comment:
should the option name be L1L2 (same as the command line arg)

Reply:
done